firewood/src/manager.rs

```diff
@@ -20,11 +20,15 @@ use metrics::gauge;
 use typed_builder::TypedBuilder;
 
 use crate::merkle::Merkle;
+#[cfg(test)]
+use crate::root_store::MockStore;
```

Comment:
nit: Move this down so it's not in the main block of use statements to improve readability